{"ast":null,"code":"export default {};","map":{"version":3,"mappings":"AACA,eAAe,EAAf","names":[],"sourceRoot":"","sources":["C:\\Users\\Max\\Desktop\\project1.0.0-main\\src\\views\\auth\\errors\\500.vue"],"sourcesContent":["<script>\nexport default {\n    \n}\n</script>\n\n<template>\n     <!-- auth-page wrapper -->\n        <div class=\"auth-page-wrapper py-5 d-flex justify-content-center align-items-center min-vh-100\">\n\n            <!-- auth-page content -->\n            <div class=\"auth-page-content overflow-hidden p-0\">\n                <div class=\"container-fluid\">\n                    <div class=\"row justify-content-center\">\n                        <div class=\"col-xl-4 text-center\">\n                            <div class=\"error-500 position-relative\">\n                                <img src=\"@/assets/images/error500.png\" alt=\"\" class=\"img-fluid error-500-img error-img\" />\n                                <h1 class=\"title text-muted\">500</h1>\n                            </div>\n                            <div>\n                                <h4>Internal Server Error!</h4>\n                                <p class=\"text-muted w-75 mx-auto\">Server Error 500. We're not exactly sure what happened, but our servers say something is wrong.</p>\n                                <router-link to=\"/\" class=\"btn btn-success\"><i class=\"mdi mdi-home me-1\"></i>Back to home</router-link>\n                            </div>\n                        </div><!-- end col-->\n                    </div>\n                    <!-- end row -->\n                </div>\n                <!-- end container -->\n            </div>\n            <!-- end auth-page content -->\n        </div>\n        <!-- end auth-page-wrapper -->\n</template>"]},"metadata":{},"sourceType":"module"}